lib/dgram.js

```diff
@@ -283,20 +283,25 @@ function fixBufferList(list) {
 function enqueue(self, toEnqueue) {
   // If the send queue hasn't been initialized yet, do it, and install an
   // event handler that flushes the send queue after binding is done.
-  if (!self._sendQueue) {
-    self._sendQueue = [];
-    self.once('listening', function() {
-      // Flush the send queue.
-      for (var i = 0; i < this._sendQueue.length; i++)
-        this.send.apply(self, this._sendQueue[i]);
-      this._sendQueue = undefined;
-    });
+  if (!self._queue) {
+    self._queue = [];
```

Comment:
why?
Plus, this is used during startup only, so I see little harm in deopting there.